components/Cards/docs/accessibility.md

```diff
@@ -0,0 +1,66 @@
+### Accessibility Labels
+
+To help ensure your Cards are accessible to as many users as possible, please be sure to review the following 
+recommendations:
+
+#### Accessibility Labels for Nested Elements
+
+Set an appropriate[`accessibility
+Label`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619577-accessibilitylabel) 
+for meaningful content that is not textual by default, like images. Also make sure to enable the element's 
+[`isAccessible
+Element`](https://developer.apple.com/documentation/objectivec/nsobject/1615141-isaccessibilityelement) 
+property so it is detectable by assistive technologies.
+
+Swift
+```swift
+myImageView.isAccessibilityElement = true
+myImageView.accessibilityLabel = "Image title or some info"
```

Comment:
I believe buttons generally do not need an accessibility labels since they have a meaningful default (the button's title).  Nested elements would need accessibility labels only in cases when they don't have meaningful defaults, like images. Obviously not all images need to be accessible, so this example refers to the case in which it is needed.